Makefile

```diff
@@ -1,41 +1,43 @@
 # several pytest settings
-DURATION ?= all  # pytest duration, one of short, long or all
+DURATION ?= short  # pytest duration, one of short, long or all
```

Comment:
since the `test.sh` uses DURATION=all, this should be default so someone can easily reproduce the failing tests without needing to pass in DURATION=all to `make test`.

Reply:
Discussed offline; leaving as all here and changing to default to all in conftest.py